build.sbt

```diff
@@ -1,34 +1,69 @@
+import sbt.Project.projectToRef
```

Comment:
This file is getting more and more complicated to read, but this is only a beginning of the project